app/assets/stylesheets/_search_results.scss

```diff
@@ -71,6 +71,11 @@
     width: 100%;
   }
 
+  &__add-to-vocab-sheet {
```

Comment:
I think this should be an element on the sign-container class rather than the card class. The card class is used almost exclusively on the search results page right now and has its own equivalent button which is positioned slightly differently, so this could get confusing in the future as an element of that.